src/BasisTheory.ts

```diff
@@ -66,10 +66,7 @@ const defaultInitOptions: Required<BasisTheoryInitOptionsWithoutElements> = {
   apiBaseUrl: DEFAULT_BASE_URL,
   elements: false,
   appInfo: {},
-  httpsAgent: {
-    keepAlive: true,
-    keepAliveMsecs: DEFAULT_KEEPALIVE_TIMEOUT_MS,
-  },
+  httpsAgent: axios.defaults.httpAgent,
```

Comment:
should this be `axios.defaults.httpsAgent`? (https not http)

Reply:
We are removing this option @dannycoates. #322

Reply:
hey @dannycoates thanks for the review! you are right, but in this scenario since it wasn't defined before in the SDK, both values would be `undefined`. this option was added experimentally and in the end we deemed unnecessary and removed as per @djejaquino comment.